.github/workflows/ci.yml

```diff
@@ -55,6 +55,9 @@ jobs:
           path: |
             **/build/reports/**
 
+      - name: Check LFS files
```

Comment:
let's check this earlier, right after the checkout step. Earlier feedback the better